src/optimum/rbln/transformers/models/decoderonly/modeling_decoderonly.py

```diff
@@ -297,8 +298,16 @@ def _get_rbln_config(
         rbln_attn_impl = rbln_kwargs.get("attn_impl", None)
         rbln_kvcache_partition_len = rbln_kwargs.get("kvcache_partition_len", None)
         rbln_quantization = QuantizationManager.validate_quantization_config(rbln_kwargs.get("quantization", None))
+        rbln_prefill_chunk_size = rbln_kwargs.get("prefill_chunk_size", 128)
+
+        if rbln_prefill_chunk_size is None:
+            warnings.warn("prefill_chunk_size is None. Falling back to default value: 128.", UserWarning)
+            rbln_prefill_chunk_size = 128
```

Comment:
No warnings for default value

```suggestion
        rbln_prefill_chunk_size = rbln_kwargs.get("prefill_chunk_size", None)

        if rbln_prefill_chunk_size is None:
            rbln_prefill_chunk_size = 128
```